docker_image_cleaner/cleaner.py

```diff
@@ -196,6 +196,23 @@ def main():
 
                 toc = time.perf_counter()
                 deleted = pruned[key]
+
+                if not deleted and kind == "images":
+                    # No dangling images to delete, still space to free.
+                    logging.info("No dangling images to prune, pruning _all_ images")
+                    tic = time.perf_counter()
+                    try:
+                        # prune again, this time with `dangling=False` filter,
+                        # which deletes _all_ images instead of just dangling ones
+                        pruned = docker_client.images.prune(filters={"dangling": False})
```

Comment:
Hmmm, based on this, I think that the default behavior is to prune both unused and dangling. And, now, we would only try prune the non-dangling unused images.

![image](https://user-images.githubusercontent.com/3837114/172898459-31531a17-f470-4f2f-b084-770e42ce3105.png)

Do you have reason to believe the default is to prune only the dangling images?